integration-cli/docker_cli_network_unix_test.go

```diff
@@ -783,6 +841,12 @@ func (s *DockerSuite) TestInspectApiMultipleNetworks(c *check.C) {
 	bridge := inspect121.NetworkSettings.Networks["bridge"]
 	c.Assert(bridge.IPAddress, checker.Equals, versionedIP)
 	c.Assert(bridge.IPAddress, checker.Equals, inspect121.NetworkSettings.IPAddress)
+
+	// cleanup
```

Comment:
I don't understand.
The test suite itself takes care of cleaning this stuff up as soon as the test is done.